Formula/rings.repl.rb

```diff
@@ -0,0 +1,17 @@
+class RingsRepl < Formula
+  desc "Rings: efficient Java/Scala library for polynomial rings"
+  homepage "http://ringsalgebra.io/"
+  url "https://raw.githubusercontent.com/PoslavskySV/rings/master/rings.repl/rings.repl", :using => :nounzip
+  version "2.0"
```

Comment:
There are a few issues with this formulae, but the biggest & the one that will need handling first is that this will need to be an actual release, not something plucked off of `master`.

Reply:
@DomT4 thanks for pointing out this issue! I have fixed this, link is now points to the actual release.